Insights DKP 2.2.1 Release Notes. (#4535)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md

When adding a new Attached Cluster to the Management Cluster, using a custom domain and TLS certificate issued by Let's Encrypt, the kube-oidc-proxy helm chart in the Attached Cluster did not complete installation and the associated pod returned an error.

> ### DKP Insights: Make resource requests and limits configureable.

Reviewer: ```suggestion
### DKP Insights: Resource requests and limits are now configurable
```

> ### DKP Insights: Make resource requests and limits configureable.

Reviewer 2: ```suggestion
### DKP Insights: Available resource requests and limits configuration
```

> Previously there was no way to configure the cpu & memory requests and limits for the Insights management and backend components. This is now fixed.

Reviewer: ```suggestion
You can now configure CPU and memory requests to use limits for the Insights management and backend components.
```
